include/irr/video/IGPUObjectFromAssetConverter.h

```diff
@@ -330,6 +331,7 @@ auto IGPUObjectFromAssetConverter::create(asset::ICPUImage** const _begin, asset
 		{
 			auto tmpBuff = m_driver->createFilledDeviceLocalGPUBufferOnDedMem(cpuimg->getBuffer()->getSize(),cpuimg->getBuffer()->getPointer());
 			m_driver->copyBufferToImage(tmpBuff.get(),gpuimg.get(),count,cpuimg->getRegions().begin());
+            //TODO mipmap generation
```

Comment:
you can distustingly call glGenerateMipmaps (DSA version) for now, and leave a note pointing to the mipmapper issue.